client/src/components/Table/Table.js

```diff
@@ -54,6 +57,10 @@ export default function CustomTable(props) {
               return (
                 <TableRow key={k} className={classes.tableBodyRow}>
                   {prop.map((prop, key) => {
+                    //This is only for deallocation of project where need _id in data but need not to show in UI.
+                    if (deallocateProject && key == 5) {
+                      return
```

Comment:
empty return ?

Reply:
yes. Dont want to code get executed based on condition mention. Same is also mention in comment line above.